custom_components/easee/sensor.py

```diff
@@ -391,6 +409,11 @@ def state_attributes(self):
                     # maybe for everything?
                     key = attr_key.replace(".", "_")
                 attrs[key] = self.get_value_from_key(attr_key)
+
+                # Dirty fix to get local time for schedule, could be moved later
```

Comment:
I don't think this is a good idea. If this is for presentation then I think HA has this built in or it should have it.
Either way we should not hardcode some attributes for conversion. Either we convert all values which are of datetime or none

Reply:
Yeah, well, I'm not sure. I was confused because Easee wants UTC time for set and returns for get, while it is shown in app and web in local time. But convert all could be a solution. How do you propose we implement it?

Reply:
I think HA does this automatically if the datetime object is in UTC or contains TX data

Reply:
It didn't work automatically at least, I first noticed I always got 2 hours difference when setting the time, and later I also saw that I got 2 hours difference when reading it back too. For instance when Easee app showed 16:00, I received 14:00. So seems like Easee under the hood operates in UTC but always shows local time which would be reasonable, and I agree in assuming HA would do something similar. But then there is something we should add to make it timezone aware or something, because straight out of the box it didn't work?

Reply:
Try to make sure all dates have tz=utc in BaseDict. I think Easee returns strings like "2020-07-18T19:12:53.385Z" which contains empty TZ.

Reply:
I'll help later today with a proposal

Reply:
I released a new library which sets the timezone to UTC. This should mean that HA picks that up and converts it to local time in the browser